BD-2683: Update campaign interaction data EA note (#6649)
braze-inc/braze-docs: _docs/_hidden/private_betas/campaign_data_retention.md

Campaign interaction data refers to how a user interacts with a campaign they received (for example, when a user opens campaign A or a user receives variant A). This data can be accessed for retargeting the moment you need it.

{% alert important %}

> Starting early 2024, unused campaign interaction data will be automatically archived for all customers according to the process outlined here. <br><br>This process will extend to Canvas interaction data in the future.

Author: ```suggestion
Starting early 2024, campaign interaction data will be available according to the process outlined here.<br><br>This process will extend to Canvas interaction data in the future.
```
